core/src/main/java/com/linecorp/armeria/server/annotation/ConsumesJson.java

```diff
@@ -21,10 +21,12 @@
 import java.lang.annotation.Target;
 
 /**
- * An alias for {@code @Consumes("application/json; charset=utf-8")}.
+ * An alias for {@code @Consumes("application/json")} and
+ * {@code @Consumes("application/json; charset=utf-8")} which is same meaning.
  */
 @Retention(RetentionPolicy.RUNTIME)
 @Target({ ElementType.TYPE, ElementType.METHOD })
+@Consumes("application/json")
 @Consumes("application/json; charset=utf-8")
```

Comment:
I'm curious if we remove the second annotation (the one with `charset=utf-8`). @whhan91, could you check if everything works fine after removing the second annotation?

Reply:
Have checked it, and we can remove `@Consumes("application/json; charset=utf-8")` and just leave `@Consumes("application/json")`.
Could you update it please?

Reply:
Done!